python/lsst/ip/isr/isrMock.py

```diff
@@ -556,14 +557,60 @@ def getExposure(self):
         ccd = exposure.getDetector()
         newCcd = ccd.rebuild()
         newCcd.clear()
+        readoutMap = {
+            'LL': ReadoutCorner.LL,
+            'LR': ReadoutCorner.LR,
+            'UR': ReadoutCorner.UR,
+            'UL': ReadoutCorner.UL,
+        }
         for amp in ccd:
             newAmp = amp.rebuild()
             newAmp.setLinearityCoeffs((0., 1., 0., 0.))
             newAmp.setLinearityType("Polynomial")
             newAmp.setGain(self.config.gain)
             newAmp.setSuspectLevel(25000.0)
             newAmp.setSaturation(32000.0)
+            readcorner = 'LL'
```

Comment:
Maybe `readoutCorner`?

Reply:
But does this need to be set here?  It's overridden below.

Reply:
I confirm `isLsstLike = False` gives the same on the branch as on main. I don't see why we need to set `rawFlipX,Y` to False but happy to do so. 

![isrMock_TrimmedRawMock_branch](https://github.com/lsst/ip_isr/assets/12816116/830b9f4f-859b-41c3-9154-a37758a16484)
![isrMock_TrimmedRawMock_main](https://github.com/lsst/ip_isr/assets/12816116/37208463-1546-4a50-9801-fe864e102472)